src/BackupExplorer/Parser/BackupParserImpl.cs

```diff
@@ -184,12 +182,13 @@ private void SetUpReplicaForReads()
             this.reliabilitySimulator.GrantReadAccess();
         }
 
-        private TransactionalReplicator Replicator
+        public TransactionalReplicator Replicator
```

Comment:
>public [](start = 8, length = 6)

Why change to public?

Reply:
Reverted back to private. Wanted to expose StatefulServiceContext to users of nuget, since partitionId and all helps in filtering out and querying for related data, as per the feedback from Isha .